bucket/vale.json

```diff
@@ -0,0 +1,19 @@
+{
+    "version": "2.10.6",
+    "description": "A syntax-aware linter for prose built with speed and extensibility in mind.",
+    "homepage": "https://docs.errata.ai/vale/about/",
+    "license": "MIT",
+    "url": "https://github.com/errata-ai/vale/releases/download/v2.10.6/vale_2.10.6_Windows_64-bit.zip",
```

Comment:
Clearly 64bit only

Reply:
Thx, updated  with instructions for 64-bit architecture only.